k2/csrc/fsa_utils_test.cu

```diff
@@ -651,4 +655,55 @@ TEST_F(StatesBatchSuiteTest, TestBackwardScores) {
   // host code?)
 }
 
+TEST_F(StatesBatchSuiteTest, TestArcScores) {
+  ContextPtr cpu = GetCpuContext();  // will use to copy data
```

Comment:
will **be** used?

Reply:
Remove the whole line as it is not used now